### ISSUE TITLE
Missing diagram for HSFA7/ HSFA6B-regulatory network-induced by drought and ABA pathway

### ISSUE THREAD

**Issue by guptapa** [PlantReactome/testing-issues#26]
A window popped up with error msg: "There was a problem while loading the diagram"

https://plantreactomedev.gramene.org/PathwayBrowser/#/R-OSA-9623703&PATH=R-OSA-8932729,R-OSA-9623744&DTAB=MT

**Comment by preecej** [comment]
Locating prior diagram json file for comparison.

**Comment by elserj** [comment]
Copied diagram/R-OSA-9623703*.json from older diagram folder and it appears to have fixed the issue on the pathwaybrowser. However, in diagnosing discovered same pathway is broken on curate.reactome.org.
Also, corresponding files for other species aren't in the same folder so, can't fix for all species yet.

Leaving open, for now.

**Comment by preecej** [comment]
Further clarification: We have patched the diagram for rice. There is a fundamental issue with the diagram in gk_central, pre-slice. Whatever the issues, the diagram does not display in RCT or on curator.reactome.org. This prevents the diagram from being generated, let alone projected, and will need to be addressed post-release.

**Comment by preecej** [comment]
This was patched on dev, but no longer. Need to re-patch for the time being, following instructions above.

Fundamental issue with diagram still underlying. Needs to be reviewed with curator.

**Comment by elserj** [comment]
Done on plantreactomedev for this release again.

**Comment by elserj** [comment]
According to email from Guanming on 3/30/22, issue is that no diagram is associated with pathway. Needs to be fixed by curators @guptapa @naithanis for next release.

**Comment by elserj** [comment]
Peter fixed on curator.reactome.org by recreating the diagrams from scratch on 1/10/23 based on what was in the old .json files.

He did point out that "response to drought" is the parent but was also a component of the diagram and so was removed.

**Comment by deustp01** [comment]
> based on what was in the old .json files.

If I'm reading right, and in case it matters, I used the curator tool to create a diagram for the pathway in gk_central from the current pathway annotations in gk_central. I did find a pathway diagram on the plant_reactome website and I approximately followed the circular layout of that old diagram. No json magic.

**Comment by elserj** [comment]
@deustp01 Sorry if that was confusing. I meant that as a workaround when Preece and I initially worked on this, we found that the diagram would show up on the site if we copied the .json files from a previous release. That of course didn't fix the actual issue and didn't work for the projected species.

For the last few releases, I have been manually copying the .json files from the previous release, but I wanted this fixed for good before this release before I got too far in the process.
